### PR TITLE
Seedtag - new js endpoint

### PR DESCRIPTION
**Pull request by ybootin**
♻️ Refactor

We are changing our endpoint to init the AMP config.
Now our AMP adapter only need the `adunitId` to initialize and reach our js endpoint.

Old integrations sill compatible

### CODE REVIEW

File: ads/vendors/seedtag.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function seedtag(global, data) {
   /*eslint "local/camelcase": 0*/
   global._seedtag_amp = {
     allowed_data: ['adunitId', 'placement', 'publisherId', 'forceCreative'],
```
Comment (ybootin) [inline ads/vendors/seedtag.js:26]: we keep them for compatibility. Ols integration will continue work with those parameters even if we don't use them

File: ads/vendors/seedtag.js [hunk 2]
```diff
@@ -34,5 +34,5 @@ export function seedtag(global, data) {
     global._seedtag_amp.allowed_data
   );
 
-  loadScript(global, 'https://t.seedtag.com/c/loader.js');
```
Comment (ybootin) [inline ads/vendors/seedtag.js:37]: we are now loading this assets from the new endpoint

File: ads/vendors/adsviu.md [hunk 3]
```diff
@@ -19,8 +19,6 @@ limitations under the License.
 ## Example
 
 ```html
-<amp-ad width="320" height="250" type="seedtag" data-adunit-id="0" data-publisher-id="0000-0000-01"
-    data-placement="inArticle">
```
Comment (ybootin) [inline ads/vendors/adsviu.md:23]: don't know why was there, probably an error from a previous PR

**Review by erwinmombay — APPROVED** [review]

**Comment by erwinmombay** [comment]
@ybootin do you mind rebasing.we had recently fixed a node issue that caused our tests to fail

**Comment by ybootin** [comment]
> @ybootin do you mind rebasing.we had recently fixed a node issue that caused our tests to fail

@erwinmombay done !

**Review by powerivq — APPROVED** [review]

**Pull request merged**